third-party/cmake/BuildLuarocks.cmake

```diff
@@ -202,7 +202,7 @@ if(USE_BUNDLED_BUSTED)
   # DEPENDS on the previous module, because Luarocks breaks if parallel.
   add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/nvim-client
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build nvim-client 0.1.0-1 ${LUAROCKS_BUILDARGS}
+    ARGS build https://raw.githubusercontent.com/phodge/lua-client/0.1.0-1/nvim-client-0.1.0-1.rockspec --branch=always-dump-corrupted-data ${LUAROCKS_BUILDARGS}
```

Comment:
sorry! The URL should be:

    https://raw.githubusercontent.com/phodge/lua-client/always-dump-corrupted-data/nvim-client-0.1.0-1.rockspec

since the `0.1.0-1` tag isn't relevant here.

Reply:
Gah I should've noticed that. No wonder all the tests are passing again! I have to go do family dinner routine now, I'll try to get back on again in a few hours and implement your feedback so that you have time for another round of reviews while I sleep. :-)